modules/ivc_champva/spec/services/pdf_filler_spec.rb

```diff
@@ -29,29 +29,29 @@
     end
   end
 
-  # describe '#generate' do
-  #   forms.each do |file_name|
-  #     context "when mapping the pdf data given JSON file: #{file_name}" do
-  #       let(:form_number) { file_name.gsub('-min', '') }
-  #       let(:expected_pdf_path) { "tmp/#{file_name}-tmp.pdf" }
-  #       let(:data) { JSON.parse(File.read("modules/ivc_champva/spec/fixtures/form_json/#{file_name}.json")) }
-  #       let(:form) { "IvcChampva::#{form_number.titleize.gsub(' ', '')}".constantize.new(data) }
+  describe '#generate' do
+    forms.each do |file_name|
+      context "when mapping the pdf data given JSON file: #{file_name}" do
+        let(:form_number) { file_name.gsub('-min', '') }
+        let(:expected_pdf_path) { "tmp/#{file_name}-tmp.pdf" }
+        let(:data) { JSON.parse(File.read("modules/ivc_champva/spec/fixtures/form_json/#{file_name}.json")) }
+        let(:form) { "IvcChampva::#{form_number.titleize.gsub(' ', '')}".constantize.new(data) }
 
-  #       after { FileUtils.rm_f(expected_pdf_path) }
+        after { FileUtils.rm_f(expected_pdf_path) }
 
-  #       context 'when a legitimate JSON payload is provided' do
-  #         it 'properly fills out the associated PDF' do
-  #           filled_pdf_path = Rails.root.join('modules', 'ivc_champva', 'spec', 'fixtures', 'pdfs',
-  #                                             "#{file_name}-filled.pdf")
+        context 'when a legitimate JSON payload is provided' do
+          it 'properly fills out the associated PDF' do
+            filled_pdf_path = Rails.root.join('modules', 'ivc_champva', 'spec', 'fixtures', 'pdfs',
+                                              "#{file_name}-filled.pdf")
 
-  #           described_class.new(form_number:, form:).generate
+            described_class.new(form_number:, form:).generate
```

Comment:
At least on the `simple_forms_api` version, this was flaky because there wasn't a unique name per PDF. Over there I believe it was solved by adding:
`let(:name) { SecureRandom.hex }` and then adding the `name` param to the generate part: `described_class.new(form_number:, form:, name:).generate`

Reply:
@bosawt Ah okay good to know.